game/src/js/scenes/TitleScene.js

```diff
@@ -1,44 +1,60 @@
 import ServiceWorker from 'util/ServiceWorker';
 import Checkbox from 'objects/ui/Checkbox';
+import Button from 'objects/ui/Button'
+
 
 export default class TitleScene extends Phaser.Scene {
 
   constructor() {
     super({key: 'TitleScene'});
   }
-
+  
   preload() {
   }
-    
+
   create() {
-    this.add.image(400, 300, 'sky');
+    this.add.image(400, 430, 'sky');
 
-    let particles = this.add.particles('red');
+    // let particles = this.add.particles('red');
 
-    let emitter = particles.createEmitter({
-      speed: 100,
-      scale: { start: 1, end: 0 },
-      blendMode: 'ADD'
-    });
+    // let emitter = particles.createEmitter({
+    //   speed: 100,
+    //   scale: { start: 1, end: 0 },
+    //   blendMode: 'ADD'
+    // });
 
     let logo = this.add.image(400, 100, 'logo');
+    //multi player button
+   
+    let multi_player_button = new Button(this, 280, 200, 'multi_player');
+   
+    multi_player_button.buttonDown(button => 
+    {
+      this.scene.start('CharacterSelectionScene', {type: 'multi_player'});
+    })
 
-    emitter.startFollow(logo);
+    // emitter.startFollow(logo);
+    //single player button
+    let single_player_button = new Button(this, 280, 320, 'single_player');
+   
+    single_player_button.buttonDown(button => 
+    {
+      this.scene.start('CharacterSelectionScene', {type: 'single_player'});
+    })
 
-    this.add.text(220, 350, 'Press ENTER to begin', {
-      font: '34px Arial',
-      fill: '#ffffff',
-      stroke: '#000000',
-      strokeThickness: '6',
-      shadowFill: '#ffffff',
-      shadowStroke: '#ffffff',
-      shadowOffsetY: '10',
-      shadowBlur: '5'
-    });
+    const text_style = {font: "bold 18px Arial", fill: "#fff", boundsAlignH: "center", backgroundColor: 'rgba(0,0,0)', boundsAlignV: "middle"};
+    
+    //controls, credits, offline mode buttons + their texts
+    let controls_button = new Button(this, 150, 433, 'controls_button');
+    let controls = this.add.text(108, 490, 'controls', text_style);
+    let credits_button = new Button(this, 275, 433, 'credits_button');
+    let credits = this.add.text(235, 490, 'credits', text_style);
+    let SW = this.add.text(355, 490, 'enable offline mode', text_style)
+  
 
     let serviceWorker = new ServiceWorker();
     
-    let checkbox = new Checkbox(this, 45, 560, 'Enable offline mode', serviceWorker.isRegistered());
+    let checkbox = new Checkbox(this, 400, 433, serviceWorker.isRegistered());
```

Comment:
looks like this is passing a boolean as the checkbox's text. The deleted line above looks right to me.

By removing that parameter and not updating the constructor on the checkbox, now the box will not be checked when offline mode is enabled.